packages/back-end/test/features.test.ts

```diff
@@ -1322,6 +1355,7 @@ describe("SDK Payloads", () => {
         environment: "production",
         groupMap: groupMap,
         experimentMap: experimentMap,
+        safeRolloutMap: safeRolloutMap,
```

Comment:
question: Do we have tests that are validating the logic of the ramp up schedule? ensuring the coverage being returned matches what we would expect from a SafeRollout.